src/main/java/com/readup/server/auth/application/OAuth2AuthenticationSuccessHandler.java

```diff
@@ -1,11 +1,51 @@
 package com.readup.server.auth.application;
 
+import static com.readup.server.auth.application.TokenProvider.*;
+
+import java.io.IOException;
+
+import org.springframework.security.core.Authentication;
 import org.springframework.security.web.authentication.SimpleUrlAuthenticationSuccessHandler;
 import org.springframework.stereotype.Component;
 
+import com.readup.server.auth.dto.AuthTokens;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.servlet.http.HttpSession;
 import lombok.RequiredArgsConstructor;
 
 @Component
 @RequiredArgsConstructor
 public class OAuth2AuthenticationSuccessHandler extends SimpleUrlAuthenticationSuccessHandler {
+
+	private final TokenProvider tokenProvider;
+	private final CookieProvider cookieProvider;
+	private static final String BASE_URI = "/";
+
+	@Override
+	public void onAuthenticationSuccess(
+		HttpServletRequest request, HttpServletResponse response, Authentication authentication) throws
+		IOException, ServletException {
+
+		HttpSession session = request.getSession(false);
+		String redirectUri = (session != null && session.getAttribute("redirect_uri") != null) ? (String)session.getAttribute("redirect_uri") : BASE_URI;
```

Comment:
```suggestion
String redirectUri = (Objects.nonNull(session) && Objects.nonNull(session.getAttribute("redirect_uri"))) ? (String)session.getAttribute("redirect_uri") : BASE_URI;
```

Objects.nonNull을 사용해 보시는건 어떨까요? 조금 더 읽기 편할 수 있을거 같아요!

Reply:
개인적으로는 != null  이렇게 쓰는게 직관적이라고 생각되어서요 
각자만의 스타일인 것 같긴 한데 Objects.nonNull 이걸로 코드 컨벤션 정하게 되는 걸까요?

Reply:
음 하나로 통일하면 좋을것 같은데요! != null or == null 로 해도 좋다고 생각합니다 :)